static/sass/_lagunita.scss

```diff
@@ -344,6 +344,18 @@ section.introduction h1.sr {
     min-height: 100px;
 }
 
+// Modal styling
```

Comment:
Where are the styling and content for this issue coming from?

Would this issue affect any theme running our platform code?
If so, would we want to include the fix in the platform styling?